### DISCUSSION TITLE
Deployment Error: node-pre-gyp WARN Hit error bad download

### DISCUSSION THREAD

**Discussion by BrOrlandi** [vercel/vercel#5138] [Help]
My deployment fails at `npm install` due to an unknown error on installing `node-pre-gyp` in `grpc`.

I did not install any new dependencies that may cause this error. It seems to be not related to my code.
I tried to redeploy a successful deployment and the same error started to happen.

The dependency I have which uses `grpc` is `@firebase/firestore`.

See details on the error below:
![image](https://user-images.githubusercontent.com/1594658/91912083-c4390500-ec88-11ea-9b4c-e37669b878cb.png)

More below:

![image](https://user-images.githubusercontent.com/1594658/91912168-f2b6e000-ec88-11ea-99db-43deeeb216e3.png)

And in the end the build fails with: 
`Error: Command "npm install" exited with 1`

**Comment by BrOrlandi** [comment]
I figure it out updating the firebase dependency. But the first error message still shows, maybe is something to be fixed.

**Reply by dylanwatsonsoftware** [reply]
Hmm do you know what firebase version you upgraded to? I'm getting this same issue.

**Reply by BrOrlandi** [reply]
To Node 14. 😉

**Reply by dylanwatsonsoftware** [reply]
Ah I haven't upgraded to the latest firebase yet.

**Comment by paulogdm** [comment]
We are receiving some reports that Node 12@latest (12.18.3 at the moment) may break some libraries. We recommend users facing such behavior to either upgrade dependencies or us Node10:

```js
// package.json
{
  "engines": {
    "node": "10.x"
  }
}
```

**Reply by dylanwatsonsoftware** [reply]
So I was getting the same issue more recently as the default node is now 14 and 10 seems to have been removed. Setting the node engine to "12.x" worked for me!
